pkg/controllers/status/cluster_status_controller.go

```diff
@@ -169,8 +169,10 @@ func (c *ClusterStatusController) syncClusterStatus(cluster *clusterv1alpha1.Clu
 
 		// get the list of APIs installed in the member cluster
 		apiEnables, err := getAPIEnablements(clusterClient)
-		if err != nil {
-			klog.Errorf("Failed to get APIs installed in Cluster %s. Error: %v.", cluster.GetName(), err)
+		if err != nil && apiEnables == nil {
+			klog.Errorf("Failed to get any APIs installed in Cluster %s. Error: %v.", cluster.GetName(), err)
+		} else if err != nil && apiEnables != nil {
+			klog.Warningf("Failed to get all APIs installed in Cluster %s. Error: %v.", cluster.GetName(), err)
 		}
```

Comment:
```suggestion
		if len(apiEnables) == 0 {
			klog.Errorf("Failed to get any APIs installed in Cluster %s. Error: %v.", cluster.GetName(), err)
		} else if err != nil {
			klog.Warningf("Maybe get partial(%d) APIs installed in Cluster %s. Error: %v.", len(apiEnables), cluster.GetName(), err)
		}
```
Does this looks better?

Reply:
I have some reservations. ` len(apiEnables) == 0` does not means a Error,because it is a result a filtered by the result of `_, apiResourceList, err := clusterClient.KubeClient.Discovery().ServerGroupsAndResources()`,see line 379 ~ 340 （apiEnables是经过过滤处理后得到的结果，如果apiResourceList 不为nil，但是经过过滤处理后apiEnables为0，这个情况是正常的，虽然这种情况k8s不太可能出现)

Reply:
> ApiEnables is the result of filtering. If the apiResourceList is not nil, but the lenth of apiEnables is 0 after filtering, this is normal, although the situation in k8S cannot happen, but it doesn't make sense logically

We just filtered sub-resources, it `should` returns a non-nil `apiEnables`, otherwise, I can assume there must be an error.

Reply:
> it `should` returns a non-nil `apiEnables`, otherwise, I can assume there must be an error.

ok,if so,line 373-375 should be deleted,and pass the result and err to the outside directly,right?
![image](https://user-images.githubusercontent.com/17697155/172334475-6d2000eb-a872-4e8c-81d7-4af9c3794151.png)

Reply:
We can keep it as a short path. It's not a big deal. Both are ok to me.